src/widget/form/settings/avform.cpp

```diff
@@ -215,7 +227,7 @@ void AVForm::onResProbingFinished(QList<QSize> res)
         bodyUI->videoModescomboBox->setCurrentIndex(savedResIndex);
     else
         bodyUI->videoModescomboBox->setCurrentIndex(bodyUI->videoModescomboBox->count()-1);
-}
+}*/
```

Comment:
Like @agilob asked. Can you please remove those commented lines? We have a version system for that.